package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.0.1",
   "private": true,
   "devDependencies": {
+    "bourbon": "^4.2.7",
```

Comment:
These should be in dependencies rather than dev dependencies, or no? I guess because the build process creates the CSS it's ok?